### DISCUSSION TITLE
What's the meaning of blockchain?

### DISCUSSION THREAD

**Discussion by popstar7** [Finalgoal231/support#8] [Q&A] [answered]
I am new in blockchain.
Please let me know what blockchain is.

**Comment by noahstoner** [comment, marked as answer]
A blockchain is “a distributed database that maintains a continuously growing list of ordered records, called blocks.” These blocks “are linked using cryptography. Each block contains a cryptographic hash of the previous block, a timestamp, and transaction data.
